packages/gatsby/babel.config.js

```diff
@@ -3,5 +3,7 @@
 // Ref: https://github.com/babel/babel/pull/7358
 module.exports = {
   sourceMaps: true,
-  presets: [["babel-preset-gatsby-package"]],
+  presets: [["babel-preset-gatsby-package", {
+    keepDynamicImports: [`./src/utils/feedback.ts`]
+  }]],
```

Comment:
this PR adds a way to mark individual files to not transform

```await import("")``` 

to 

```
await Promise.resolve().then(() => _interopRequireWildcard(require("")));
```

and instead keep dynamic import

We have plenty of dynamic imports in our codebase / `gatsby` package and switching it everywhere (through already existing `esm: true` option that `babel-preset-gatsby-package`) doesn't seem like good idea and definitely would require solid testing.

This approach allows us to address individual modules and we can over-time migrate more things if needed